src/main/resources/native/linux/launcher.c

```diff
@@ -36,6 +49,11 @@ int main(int argc, const char* argv[]) {
     (*run_main)(argc, argv);
 }
 
+float powwrapper(float a, float b) {
```

Comment:
maybe move this up? right after `__wrap_pow`?

Reply:
that shouldn't be there at all -- I removed it.